.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: SAFE Template release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
```

Comment:
We probably need `dotnet tool restore` back here.

Reply:
@theprash done. Otherwise this PR looks ready to merge to me